### PR TITLE
Sturtup improvements

### PR DESCRIPTION
**Pull request by DmitrijOkeanij**
Refactor some sturtup settings and behavior
- Improved "help" command text
- Added test if config dirrectory does not exists
- Remaked log meesages to console messages in some places (it needed because in this cases it must be directly Console information, not logging to db or others)

### CODE REVIEW

File: SunEngine.Cli/StartupConfiguration.cs [hunk 1]
```diff
@@ -68,20 +72,41 @@ private string GetConfigurationDirectory(IEnumerable<string> arguments)
...
+            Console.WriteLine($"Configuration directory \"{configurationFileName}\" will be used.");
             return configurationFileName;
         }
+
+        public void ExitApplicationIfConfigurationIsNotValid()
+        {
+            bool failed = !TestIfConfigurationDirectoryExists(ConfigurationDirectoryRoute);
+
+            if(failed)
+                Environment.Exit(0);
```
Comment (PutienkoEN) [inline SunEngine.Cli/StartupConfiguration.cs]: This class stores configuration, why we use Environment.Exit(0); in it?
Environment.Exit(0); should be used in Program class, not here. It's an abstraction violation.

Try to add unit test for this method.

Comment (DmitrijOkeanij) [reply SunEngine.Cli/StartupConfiguration.cs]: I refactor this. Check now.

File: SunEngine.Cli/StartupConfiguration.cs [hunk 2]
```diff
@@ -12,17 +13,19 @@ public class StartupConfiguration
     {
         private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
         
-        private const string ConfigurationArgumentName = "config:";
-        private const string DefaultConfigurationFileName = "Config";
+        public const string ConfigurationArgumentName = "config:";
+        public const string DefaultConfigurationFileName = "Config";
+        public const string ConfigurationDirectoryNameEnd = ".Config";
+
+        public const string HelpCommand = "help";
```
Comment (PutienkoEN) [inline SunEngine.Cli/StartupConfiguration.cs:18]: Why these variables should be public? It'll break abstraction.

File: SunEngine.Cli/StartupConfiguration.cs [hunk 3]
```diff
@@ -12,17 +13,19 @@ public class StartupConfiguration
     {
         private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
```
Comment (PutienkoEN) [inline SunEngine.Cli/StartupConfiguration.cs]: We can remove it now I believe.

**Pull request merged**